deployments/yunikorn-application/application-definition.yaml

```diff
@@ -50,14 +50,40 @@ spec:
         spec:
           type: object
           properties:
-            minMember:
-              type: integer
+            appId:
+              type: string
+            policy:
+              type: object
+              properties:
+                policy:
+                  type: string
+                timeout:
+                  type: integer
+                retrySecond:
+                  type: integer
```

Comment:
I think this should be a map that supports arbitrary key-value pairs.
because different policies will have different parameter names, and values